mltsp/celery_task_tools.py

```diff
@@ -7,43 +9,49 @@
 import csv
 
 
-def create_and_pickle_model(data_dict, featureset_key, model_type,
-                            in_docker_container):
+def create_and_pickle_model(data_dict, model_key, model_type,
+                            model_options):
     """Create scikit-learn RFC model object and save it to disk.
 
     Parameters
     ----------
     data_dict : dict
         Dictionary containing features data (key 'features') and
-        class list (key 'classes').
+        targets list (key 'targets').
     featureset_key : str
         RethinkDB ID of associated feature set.
     model_type : str
-        Abbreviation of the type of classifier to be created.
-    in_docker_container : bool
-        Boolean indicating whether function is being called from within
-        a Docker container.
+        Abbreviation of the type of model to be created.
+    model_options : dict, optional
+        Dictionary specifying `scikit-learn` model parameters to be used.
 
     """
-    # Build the model:
-    # Initialize
-    ntrees = 1000
-    njobs = -1
-    rf_fit = RFC(n_estimators=ntrees, max_features='auto', n_jobs=njobs)
-    print("Model initialized.")
+
+    if "RF" in model_type and "n_estimators" not in model_options:
```

Comment:
Do we need these default options here, or should we have the front-end do the parameter validation, or just pass whatever into sklearn and see what happens? I could see an argument for either way, this just looks a bit strange to me to have a couple of hard-coded defaults specific to these models...